UI/pause_menu.tscn

```diff
@@ -0,0 +1,50 @@
+[gd_scene load_steps=2 format=3 uid="uid://xltm3j142qwi"]
+
+[ext_resource type="Script" path="res://UI/pause_menu.gd" id="1_2350e"]
+
+[node name="PauseMenu" type="Control"]
+process_mode = 2
+layout_mode = 3
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+script = ExtResource("1_2350e")
+
+[node name="Panel" type="Panel" parent="."]
+layout_mode = 1
+anchors_preset = 8
+anchor_left = 0.5
+anchor_top = 0.5
+anchor_right = 0.5
+anchor_bottom = 0.5
+offset_left = -127.0
+offset_top = -160.0
+offset_right = 123.0
+offset_bottom = 158.0
+grow_horizontal = 2
+grow_vertical = 2
```

Comment:
Why isn't this using the entire screen?

Reply:
the `control` above is using the entire screen and the `panel` is just centered

Reply:
Well, since we don't have designs yet... 🤷‍♂️